watertap/examples/flowsheets/MD/MD_single_stage_continuous_recirculation.py

```diff
@@ -0,0 +1,662 @@
+    )
+    m.fs.properties_cold_ch.set_default_scaling(
+        "flow_mass_phase_comp", 1e2, index=("Liq", "TDS")
+    )
+    m.fs.properties_vapor.set_default_scaling(
+        "flow_mass_phase_comp", 1, index=("Liq", "H2O")
+    )
+    m.fs.properties_vapor.set_default_scaling(
+        "flow_mass_phase_comp", 1, index=("Vap", "H2O")
+    )
+    # set unit model values
+    iscale.set_scaling_factor(m.fs.pump_permeate.control_volume.work, 1e-3)
+    iscale.set_scaling_factor(m.fs.pump_brine.control_volume.work, 1e-3)
+    iscale.set_scaling_factor(m.fs.pump_feed.control_volume.work, 1e-3)
+    m.fs.feed.properties[0].flow_vol_phase["Liq"]
+    m.fs.feed.properties[0].mass_frac_phase_comp["Liq", "TDS"]
+
+    iscale.set_scaling_factor(m.fs.hx.hot.heat, 1e-3)
+    iscale.set_scaling_factor(m.fs.hx.cold.heat, 1e-3)
+    iscale.set_scaling_factor(m.fs.hx.overall_heat_transfer_coefficient, 1e-3)
+    iscale.set_scaling_factor(m.fs.hx.area, 1e-1)
+    iscale.set_scaling_factor(m.fs.MD.hot_ch.heat, 1e-3)
+    iscale.set_scaling_factor(m.fs.MD.cold_ch.heat, 1e-3)
+    iscale.set_scaling_factor(m.fs.heater.control_volume.heat, 1e-3)
+    iscale.set_scaling_factor(m.fs.chiller.control_volume.heat, 1e-3)
+    iscale.set_scaling_factor(m.fs.MD.area, 1e-1)
+
+    iscale.calculate_scaling_factors(m)
+
+    return m
+
+
+def add_costs(m):
+    m.fs.costing = WaterTAPCosting()
+    m.fs.MD.costing = UnitModelCostingBlock(flowsheet_costing_block=m.fs.costing)
+    m.fs.heater.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=cost_heater_chiller,
+        costing_method_arguments={"HC_type": "electric_heater"},
+    )
+    m.fs.chiller.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=cost_heater_chiller,
+        costing_method_arguments={"HC_type": "chiller"},
+    )
+    m.fs.pump_feed.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=cost_pump,
+        costing_method_arguments={"pump_type": "low_pressure"},
+    )
+    m.fs.pump_permeate.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=cost_pump,
+        costing_method_arguments={"pump_type": "low_pressure"},
+    )
+    m.fs.pump_brine.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=cost_pump,
+        costing_method_arguments={"pump_type": "low_pressure"},
+    )
+    m.fs.hx.costing = UnitModelCostingBlock(flowsheet_costing_block=m.fs.costing)
+    m.fs.mixer.costing = UnitModelCostingBlock(flowsheet_costing_block=m.fs.costing)
+
+    m.fs.costing.cost_process()
+    m.fs.costing.add_annual_water_production(m.fs.permeate.properties[0].flow_vol)
+    m.fs.costing.add_specific_energy_consumption(m.fs.permeate.properties[0].flow_vol)
+    m.fs.costing.add_LCOW(m.fs.permeate.properties[0].flow_vol)
+
+
+def set_operating_conditions(m):
```

Comment:
When setting the operating conditions, it would be nice to have the units commented next to the variables for readability.  This is a nice to have but not need to have.

Reply:
Addressed